src/test/groovy/graphql/schema/diffing/ana/EditOperationAnalyzerTest.groovy

```diff
@@ -2035,6 +2035,94 @@ class EditOperationAnalyzerTest extends Specification {
         changes.objectDifferences["Node"] instanceof ObjectDeletion
     }
 
+    def "directive deleted with argument"() {
+        given:
+        def oldSdl = '''
+        type Query {
+            node: String
+        }
+        directive @test(message: String) on FIELD
+        '''
+        def newSdl = '''
+        type Query {
+            node: String
+        }
+        '''
+
+        when:
+        def changes = calcDiff(oldSdl, newSdl)
+
+        then:
+        changes.directiveDifferences["test"] instanceof DirectiveDeletion
+    }
+
+    def "interface added with field argument"() {
```

Comment:
These are duplicates of the directive deleted tests, but with the old and new schema swapped and the check asserts there's a `XAddition` difference in the result.